src/main/java/com/powsybl/openloadflow/network/impl/PropagatedContingency.java

```diff
@@ -399,12 +406,15 @@ private ContingencyConnectivityLossImpact findBusesAndBranchesImpactedBecauseOfC
                     .filter(LfBranch::isConnectedAtBothSides)
                     .forEach(connectivity::removeEdge);
 
-            if (connectivity.getConnectedComponent(network.getSlackBus()).size() == 1) {
-                // FIXME
-                // If a contingency leads to an isolated slack bus, this bus is considered as the main component.
-                // In that case, we have an issue with a different number of variables and equations.
-                LOGGER.error("Contingency '{}' leads to an isolated slack bus: not supported", contingency.getId());
-                return new ContingencyConnectivityLossImpact(false, 0, Collections.emptySet(), Collections.emptySet());
+            if (isSlackBusIsolated(connectivity, network.getSlackBus())) {
```

Comment:
```suggestion
            if (network.getSlackBuses().stream().anyMatch(slackBus -> isSlackBusIsolated(connectivity, slackBus))) {
```

fixes a modified version of `testBusBarSectionContingencyWithSlackBusRelocation()` with following parameters:
```java
lfParameters.getExtension(OpenLoadFlowParameters.class).setSlackBusesIds(List.of("VL1_1", "VL1_0")).setMaxSlackBusCount(2);
```

but remains an issue to solve, i am getting `Expected to have same number of equations (9) and variables (10)` with
```java
lfParameters.getExtension(OpenLoadFlowParameters.class).setSlackBusesIds(List.of("VL1_0")).setMaxSlackBusCount(2);
```

for N situ:
```
INFO  14:26:59.050 c.p.o.n.LfNetwork    | Network {CC0 SC0}, slack buses are [VL1_0] (method='Parameter bus')
INFO  14:26:59.051 c.p.o.n.LfNetwork    | Network {CC0 SC0}, reference bus is VL1_0 (method='First slack')
```

and CTG situation:
```
INFO  14:26:59.216 c.p.o.n.LfNetwork    | Network {CC0 SC0}, slack buses are [VL1_1, VL2_0] (method='Most meshed bus')
TRACE 14:26:59.217 c.p.o.n.LfNetworkListenerTracer | onSlackBusChange(bus=VL1_1, slack=true)
TRACE 14:26:59.217 c.p.o.n.LfNetworkListenerTracer | onSlackBusChange(bus=VL2_0, slack=true)
INFO  14:26:59.217 c.p.o.n.LfNetwork    | Network {CC0 SC0}, reference bus is VL1_1 (method='First slack')
TRACE 14:26:59.217 c.p.o.n.LfNetworkListenerTracer | onReferenceBusChange(bus=VL1_1, reference=true)
...

java.util.concurrent.CompletionException: com.powsybl.commons.PowsyblException: Expected to have same number of equations (9) and variables (10)
```

Reply:
probably a weird edge case where with provided settings we get more slack buses in N-1 situation compared to N situation